tests/suite/zqd/services.sh

```diff
@@ -28,7 +33,7 @@ export AWS_ACCESS_KEY_ID=minioadmin
 export AWS_SECRET_ACCESS_KEY=minioadmin
 export AWS_S3_ENDPOINT=http://localhost:$(cat $portdir/minio)
 
-zqd listen -l=localhost:0 -portfile="$portdir/zqd" -datadir=./zqdroot &
+zqd listen -l=localhost:0 -portfile="$portdir/zqd" -datadir=$zqdroot 2> zqd.error &
```

Comment:
```suggestion
zqd listen -l=localhost:0 -portfile="$portdir/zqd" -datadir="$zqdroot" 2> zqd.error &
```
And `2> zqd.error` feels like it's hiding errors in a non-obvious place. Is it necessary?

Reply:
I put this in because zqd by default writes logs to stderr which messes up some of my tests for checking the output of stderr. I suppose I could change the zqd logger.Level to "warn"? Thoughts?

Reply:
I think changing the logging level is the way to go, and Warn strikes me as appropriate.